packages/frontend-shared/package.json

```diff
@@ -44,7 +44,7 @@
     "rimraf": "3.0.2",
     "unplugin-icons": "^0.11.4",
     "unplugin-vue-components": "^0.15.4",
-    "vite": "^2.5.10",
+    "vite": "2.4.4",
```

Comment:
@JessicaSachs @lmiller1990 I had to pin the version of Vite as it was installing two different versions of Vite. It worked fine if you launched from launchpad but if you ran `yarn dev` from the root then it would fail for different type errors.